src/main.js

```diff
@@ -196,4 +198,18 @@ function displayPoster(){
   mainQuote.innerText = currentPoster.quote;
 };
 
+function showSaved() {
+  var allSavedPosters = "";
+  for (var i = 0; i < savedPosters.length; i++) {
+    var savedPosterHTML = `
+    <article class="mini-poster">
```

Comment:
Rather than writing in raw HTML here, can we use the lesson from this morning to apply CSS elements directly to these mini-posters? I notice that there are already some presets for us in the CSS stylesheet.

Reply:
`.mini-poster {
  background: black;
  color: white;
  height: 340px;
  padding: 25px;
  width: 425px;
}

.mini-poster img {
  height:200px;
  width: 300px;
}

.mini-poster h2,
.mini-poster h4,
.delete {
  color: white;
  line-height: 1;
  width: 90%;
}

.mini-poster h2 {
  font-size: 30px;
  font-weight: normal;
  letter-spacing: 5px;
  margin-bottom: 10px;
  text-transform: uppercase;
}`

Like, can we do this just by adding the "mini-poster" style tag to the saved posters, somehow?

Reply:
Thinking about how we turned that box pink...

Reply:
Great call on using the updated HTML tags an classes!!!! Updated  it with a new branch -- I'll close this request.